src/main/java/com/revature/domain/EvalType.java

```diff
@@ -0,0 +1,70 @@
+package com.revature.domain;
+
+import java.io.Serializable;
+import java.util.Objects;
+import javax.persistence.*;
+
+@Entity
+@Table(name="ie_eval_type")
+public class EvalType implements Serializable {
+
+	@Id
+	@Column(name="et_id")
+	private Integer id;
+	
+	@Column(name="et_description")
+	private String description;
+	
+	public EvalType() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)